client/v2/client.go

```diff
@@ -218,6 +218,31 @@ func (p *Point) PrecisionString(precison string) string {
 	return p.pt.PrecisionString(precison)
 }
 
+// Name returns the measurement name of the point
+func (p *Point) Name() string {
+	return p.pt.Name()
+}
+
+// Name returns the tags associated with the point
+func (p *Point) Tags() map[string]string {
```

Comment:
Does this return a copy of the tags, or the tags themselves? If the latter, it is mutable by the caller.